### ISSUE TITLE
A Javascript error occurred in the main process on refresh listing with keyboard shortcuts

### ISSUE THREAD

**Issue by anselmodev** [mingo-app/mingo#508]
When updating the listing with F5 or CTRL+L, an error alert is triggered and the listing is not updated.
- Mingo 1.10.0
- Running on Pop!_OS 22.04 LTS

Screenshot:
![Screenshot from 2022-11-28 16-56-14](https://user-images.githubusercontent.com/14978874/204370393-44d75843-e797-4bc9-a837-3f5d3ef327eb.png)

**Comment by tothradoslav** [comment]
Thank you. Is this happening always, or just started after some time? Also, does this happen on all collections or just a specific one? Does restarting Mingo help?

Sorry for many questions, just trying to pinpoint the possible problem.

**Comment by anselmoalima** [comment]
Hi!
This error happened a few times. I realized that even closing Mingo, the F5 or CTRL+R command kept alerting the same error. It only got back to normal after restarting my machine. Maybe it's an isolated case?

**Comment by tothradoslav** [comment]
This seems to be a problem with the code looking for an already-not-existing window. It's weird that it only got corrected after machine restart. Could you let me know if this happens again or if you have any other details about when this might be happening? Thank you!

**Comment by tothradoslav** [comment]
@anselmodev also, could it be, that you had multiple Mingo's running? Or that you had one crashing and then running a new one? Just let me know if this was the case. Thanks.

**Comment by anselmoalima** [comment]
@tothradoslav Yes, sometimes I open two or three Mingo windows connected to different databases. And at that moment I have two windows open and running normally and the reported error no longer happened.

**Comment by tothradoslav** [comment]
Ok. I will close this for now but please feel free to reopen it or send a new issue again if you experience it again. Thanks.
